manim/mobject/types/vectorized_mobject.py

```diff
@@ -879,6 +880,49 @@ def get_subcurve(self, a, b):
         vmob.pointwise_become_partial(self, a, b)
         return vmob
 
+    def get_direction(self):
+        """
+        Uses shoelace in space_ops.py to calculate the direction.
+        The direction of points determines in which direction the
+        object is drawn, clockwise or counterclockwise.
+
+        Returns
+        -------
+        :class:`str`
+            Either `"CW"` or `"CCW"`.
+        """
+        return shoelace_direction(self.get_start_anchors())
+
+    def reverse_direction(self):
+        """
+        Reverts the point direction by inverting the point order.
+
```

Comment:
Wishlish, non-blocking: I think a rendered example would be pretty cool here; maybe something like you had on your other PR?

In case you want to try to add a small example, have a look at the documentation of `manim_directive.py` to see how to use the manim directive for rendering a video in a docstring.

Reply:
I was aware that something like this was planned, but not that it's already possible, in that case I want this as well.
I take it this would go into... The Example Gallery, under Animations. I'll have a look.

Reply:
I am somewhat inclined to have it directly here in this docstring, in the reference manual.

Better connectivity and a cleaner separation between contents of the example gallery and the reference manual is something we are working on. See #591. :-)